specification/storagecache/resource-manager/readme.md

```diff
@@ -4,40 +4,51 @@
 
 This is the AutoRest configuration file for storagecache RP.
 
-
-
 ---
+
 ## Getting Started
+
 To build the SDK for storagecache, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the StorageCache API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-03-01
+tag: package-2020-10
 add-credentials: true
 title: StorageCacheManagementClient
 ```
+
+
+### Tag: package-2020-10
+
+These settings apply only when `--tag=package-2020-10` is specified on the command line.
+
+```yaml $(tag) == 'package-2020-10'
+input-file:
+  - Microsoft.StorageCache/stable/2020-10-01/storagecache.json
+```
 ### Tag: package-2020-03-01
 
 These settings apply only when `--tag=package-2020-03-01` is specified on the command line.
 
-```yaml $(tag) == 'package-2020-03-01'
+``` yaml $(tag) == 'package-2020-03-01'
```

Comment:
small cosmetic change for "space"

Their CI task fails to pickup the tag otherwise.

Reply:
Fixed in next commit.

Reply:
Fixed in next commit.

Reply:
Doesn't look like this was fixed in the latest commit.

```suggestion
```yaml $(tag) == 'package-2020-03-01'
```

Reply:
Are the spaces being fixed in these tags? Latest commits still show the space there, but not sure if that's now intended. @romahamu @brpanask

Reply:
Sounds like the space was added by autogen, and other readmes similarly have the space. Resolving.